packages/playground/src/dashboard/transfer_view.vue

```diff
@@ -19,7 +19,7 @@
                 :rules="[
                   validators.required('Recipient Twin ID is required'),
                   validators.isNotEmpty('Invalid Twin ID'),
-                  validators.isNumeric('Twin ID should be a number'),
+                  validators.isInt('Twin ID should be a valid integer.'),
                   validators.min('Twin ID should be more than 0', 1),
```

Comment:
Can we also change that to be `greater than zero` instead of `more than 0`

Reply:
done